sql/rpl_gtid.cc

```diff
@@ -4067,3 +4119,17 @@ my_bool Intersecting_gtid_event_filter::has_finished()
   }
   return FALSE;
 }
+
+my_bool Intersecting_gtid_event_filter::verify_completed_state()
+{
+  my_bool is_any_incomplete= FALSE;
+  Gtid_event_filter *subfilter;
+  for (size_t i= 0; i < m_filters.elements; i++)
```

Comment:
This method was based on `::has_finished()`.
Besides formatting, a difference is that this uses `size_t` to match the type of `m_filters.elements`.
The former had a `ulong`; it’s a potential bug, but I’m yet to review which earliest (supported) version this fix should apply on.

Reply:
Or can I just leave this to [[MDEV-26896] Enable -Wconversion globally](https://jira.mariadb.org/browse/MDEV-26896)?